### PR TITLE
Removing ActiveIssue attributes. Adding component name to all WebClient tests.

### LINKED ISSUE #18680: Validation missing when adding certain headers to WebClient.Headers collection
While investigating #17882, I discovered that adding certain request headers to `WebClient` are not error-checked the same between .NET Core and .NET Framework implementations.

Adding 'Connection', or 'Expect' request headers throws WebException when sending the request using .NET Framework.  But .NET Core does not throw exception.

Adding 'Host' request header with an illegal DNS name throws WebException when sending the request using .NET Framework.  But .NET Core does not throw exception.

Added new tests to validate this and marked the tests as disabled for .NET Core.

### LINKED ISSUE #20141: UWP: System.Net.WebClient
Tracking UWP work for System.Net.WebClient.

### PR DESCRIPTION
**Pull request by CIPop**
Fixes #20141 
Fixes #18680 (removing ActiveIssue)

### CODE REVIEW

File: src/System.Net.WebClient/tests/WebClientTest.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ namespace System.Net.Tests
     public class WebClientTest
     {
         [Fact]
-        public static void DefaultCtor_PropertiesReturnExpectedValues()
+        public static void WebClient_DefaultCtor_PropertiesReturnExpectedValues()
```
Comment (davidsh) [inline src/System.Net.WebClient/tests/WebClientTest.cs]: Why did you change the test names to prefix them with the classname.  We don't do that on any of our other test classes/methods.

 The class is the module under test, i.e. "WebClient".  And the first part of the test name is the method or property under test.  So, that would be Ctor or Properties or DownloadData etc.

Comment (CIPop) [inline src/System.Net.WebClient/tests/WebClientTest.cs]: I remember that some of the CI reports were not including the class name. If that not true, I can revert this change.

Comment (davidsh) [inline src/System.Net.WebClient/tests/WebClientTest.cs]: Because we have tests, in System.Net.Http, for example, where HttpClientHandlerTest.cs doesn't use class name in the test method name:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

If the CI system is having a problem reporting the full namespace/class/method for the tests, we should get that fixed.  cc: @safern

Comment (safern) [inline src/System.Net.WebClient/tests/WebClientTest.cs]: Do you have an occurrence of where a report is not including the full method name? This is done by XUnit and it always logs everything as Namespace.Class.Method, so you shouldn't include the class name in the test name as it will be duplicated in the logs and that is not the guidelines we follow.

**Comment by CIPop** [comment]
> Waiting for test case names to be reverted back to the original ones.

Changed the test names back to the original ones. Will open CI/Helix bugs if I see the problem I've originally described again.
